src/signals-unix.c

```diff
@@ -668,7 +668,7 @@ static void *signal_listener(void *arg)
             if (critical) {
                 bt_size += rec_backtrace_ctx(bt_data + bt_size,
                         JL_MAX_BT_SIZE / jl_n_threads - 1,
-                        signal_context);
+                        signal_context, 0);
```

Comment:
Over at #33277 I made the critical error printing aware of interpreter frames (printing is marked as JL_NOTSAFEPOINT and passes the static analyzer). So hopefully we could set this `0` to `1` there.

Reply:
There’s no gc root for that value here

Reply:
Oh I see, I forgot this version of `bt_data` is not rooted and other threads may run the GC.

If `doexit`, we could just stop the world to print this backtrace data because we're about to attempt to abort anyway and a perf hit is the least of our worries?

It's a little extra complexity though.